### PR TITLE
HRINT-2920 [Demo] Highlight Demos (Java)

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

**Comment by Danielle9897** [comment]
@marcin pls review

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsCustomizedTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        runParams.setFragmentLength(100);
+        runParams.setFragmentCount(1);
+        runParams.setTag1("+++");
+        runParams.setTag2("+++");
+        runParams.setTag3("<<<");
+        runParams.setTag4(">>>");
+
+        List<HighlightQueryResultsCustomized.DataToShow> results = new HighlightQueryResultsCustomized().run(runParams);
+        Assert.assertTrue(results.size() == 5);
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsCustomizedTest.java:26]: same here

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsCustomizedTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+        DocumentStoreHolder.store.executeIndex(new HighlightQueryResultsCustomized.EmployeesDetails());
+        HighlightQueryResultsCustomized.RunParams runParams = new HighlightQueryResultsCustomized.RunParams();
+
+        runParams.setFragmentLength(100);
+        runParams.setFragmentCount(1);
+        runParams.setTag1("+++");
+        runParams.setTag2("+++");
+        runParams.setTag3("<<<");
+        runParams.setTag4(">>>");
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsCustomizedTest.java:23]: do you set here params that are defult anyway?

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsMapReduceTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        HighlightQueryResultsMapReduce.RunParams runParams = new HighlightQueryResultsMapReduce.RunParams();
+
+        runParams.setSearchTerm("smile");
+        runParams.setPreTag("(:");
+        runParams.setPostTag(":)");
+        runParams.setFragmentLength(80);
+        runParams.setFragmentCount(1);
+
+        List<HighlightQueryResultsMapReduce.DataToShow> results = new HighlightQueryResultsMapReduce().run(runParams);
+        Assert.assertTrue(results.size() == 13);
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsMapReduceTest.java:24]: same here.

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsMapReduceTest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    @Test
+    public void test() {
+        DocumentStoreHolder.mediaStore.executeIndex(new HighlightQueryResultsMapReduce.ArtistsAllSongs());
+        HighlightQueryResultsMapReduce.RunParams runParams = new HighlightQueryResultsMapReduce.RunParams();
+
+        runParams.setSearchTerm("smile");
+        runParams.setPreTag("(:");
+        runParams.setPostTag(":)");
+        runParams.setFragmentLength(80);
+        runParams.setFragmentCount(1);
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsMapReduceTest.java:21]: same here

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsCustomizedTest.java (hunk 1 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsCustomizedTest.java:26]: using `assertEquals`

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsMapReduceTest.java (hunk 3 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsMapReduceTest.java:24]: using `assertEquals`

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsCustomizedTest.java (hunk 2 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsCustomizedTest.java:23]: removed

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsMapReduceTest.java (hunk 4 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/textSearch/HighlightQueryResultsMapReduceTest.java:21]: removed

**Comment by Danielle9897** [comment]
For issue:
https://issues.hibernatingrhinos.com/issue/HRINT-2920/Demo-Highlight-Demos-Java

**Pull request merged**